Testscripts/Windows/LIS-INSTALL-SCENARIOS.ps1

```diff
@@ -491,6 +491,12 @@ Function Main {
         }
         #PROVISION VMS FOR LISA WILL ENABLE ROOT USER AND WILL MAKE ENABLE PASSWORDLESS AUTHENTICATION ACROSS ALL VMS IN SAME HOSTED SERVICE.
         Provision-VMsForLisa -allVMData $AllVMData -installPackagesOnRoleNames "none"
+        $sts = Run-LinuxCmd -ip $AllVMData.PublicIP -port $AllVMData.SSHPort -username "root" -password $password `
```

Comment:
"||" ?

Reply:
Yes, it's ||. If hv_vmbus module is not found, then echo 1.